README.md

```diff
@@ -2,6 +2,8 @@
 
 ## Table of Contents
 
+- [Mobility Data Specification](#mobility-data-specification)
```

Comment:
To keep the TOC the same format throughout MDS, the standard we use is don't include the page name or the TOC in the TOC.

Reply:
37eaa47